<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700; color:#aa0000;&quot;&gt;Indstil Præferencer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>511</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="daytime">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Ugedage og tidsrum&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="monday">
        <property name="text">
         <string>Mandag</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="timeEditMonStart"/>
      </item>
      <item>
       <widget class="QTimeEdit" name="timeEditMonEnd"/>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="tuesday">
        <property name="text">
         <string>Tirsdag</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="timeEditTuesStart"/>
      </item>
      <item>
       <widget class="QTimeEdit" name="timeEditTuesEnd"/>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="wednesday">
        <property name="text">
         <string>Onsdag</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="timeEditWedStart"/>
      </item>
      <item>
       <widget class="QTimeEdit" name="timeEditWedEnd"/>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QCheckBox" name="thursday">
        <property name="text">
         <string>Torsdag</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="timeEditThurStart"/>
      </item>
      <item>
       <widget class="QTimeEdit" name="timeEditThurEnd"/>
      </item>
     </layout>
    </item>
    <item row="4" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QCheckBox" name="friday">
        <property name="text">
         <string>Fredag</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="timeEditFriStart"/>
      </item>
      <item>
       <widget class="QTimeEdit" name="timeEditFriEnd"/>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="locationtype">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Locationstype&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="locationType">
      <item>
       <property name="text">
        <string>Online</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fysisk</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Begge</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="6" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPlainTextEdit" name="plainTextEditRequirements"/>
      </item>
     </layout>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="requires">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Faciliteter&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="kursus">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Vælg kursus&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CourseList">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>361</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOk">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>420</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Gem</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>420</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Anuller</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClear">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Nulstil</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>70</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Resultat&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEditResult">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>110</y>
     <width>231</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
